src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,116 @@
 package core.basesyntax;
 
+import static java.lang.Math.abs;
+
+import java.util.HashSet;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private HashObject<K, V>[] hashMap;
```

Comment:
The `HashObject` class should be parameterized. According to the checklist: 'Remember to parameterize the Node class'.